app/controllers/v0/evss_claims_controller.rb

```diff
@@ -38,6 +38,18 @@ def request_decision
       render_job_id(jid)
```

Comment:
We can't change this file.
This belongs to the VA.gov folks.  It's not Lighthouse Benefits ClaimsApi stuff.